playback/src/convert.rs

```diff
@@ -23,14 +23,15 @@ pub struct Converter {
 
 impl Converter {
     pub fn new(dither_config: Option<DithererBuilder>) -> Self {
-        if let Some(ref ditherer_builder) = dither_config {
-            let ditherer = (ditherer_builder)();
-            info!("Converting with ditherer: {}", ditherer.name());
-            Self {
-                ditherer: Some(ditherer),
+        match dither_config {
```

Comment:
I don't know, where does it say that? `match` is well-suited when a value can be many variants, which is not the case here.

I don't object to merging this now it's here, but I'd hope not to ruffle up the rest of the code -- I don't think that carries much benefit.

Reply:
I guess my mind set was since it's an `Option` so you should use match when possible. You've ragged on me in the past for doing the exact same thing (using if Some instead of match for Options).

Reply:
Yeah I might have. I guess I just try to do what's the most readable. I think both can work here.

Reply:
If the "if" drops though (as in no else) I can see using "if Some" but if it's an "if, else" situation I think you should use match with an Option.

Reply:
Revert or not. I really don't care either way. It's purely a matter of style really. Functionally it makes no difference. It would be nice to always stick to a certain way in cases like this though. I will leave it up to you what you prefer though.

Reply:
Ok. I'm going to say not then?